src/index.ts

```diff
@@ -43,13 +43,25 @@ export async function registerLoginHandler(newLoginHandler: LoginHandler) {
 }
 
 export function startTestBox(config?: TestBoxConfig) {
+  // Verifications
   if (tbxStarted) {
+    info("SDK already started");
     return;
   }
 
   window.__tbxConfig = config;
-  window.__tbxLoginEvent = null;
+  info("Starting SDK...");
 
+  if (window.__tbxExtensionActive && !config.startedByExtension) {
+    // Checking for Extension
+    info(
+      "Extension is Active and SDK start had a different origin. Blocking SDK start"
+    );
+    return;
+  }
+
+  // Handlers
+  window.__tbxLoginEvent = null;
```

Comment:
What motivated this cleanup?

Reply:
You mean `window.__tbxLoginEvent = null;`? This was already being done on previous line 51, I just moved it to after the verification of the extension.